pypeit/core/pca.py

```diff
@@ -295,11 +301,10 @@ def fit_pca_coefficients(coeff, order, ivar=None, weights=None, function='legend
     # Return arrays that match the shape of the input data
     if coeff.ndim == 1:
         return np.invert(coeff_used)[0], fit_coeff[0], minx, maxx
-    return np.invert(coeff_used), fit_coeff, minx, maxx
+    return np.invert(coeff_used), fit_coeff, minx, maxx, pypeitFits
```

Comment:
I don't think you need to return the fit_coeff and the minx,maxx now that you are returning the pypeitFits object which contains those attributest.

Reply:
@kbwestfall needs to sort this issue out to write out the PypeItFits data container rather than attributest of it to disk for the PCA in edgetrace etc.